cmd/activator/main.go

```diff
@@ -229,24 +223,8 @@ func main() {
 	a := activator.NewRevisionActivator(kubeClient, servingClient, logger)
 	a = activator.NewDedupingActivator(a)
 
-	// Retry on 503's for up to 60 seconds. The reason is there is
-	// a small delay for k8s to include the ready IP in service.
-	// https://github.com/knative/serving/issues/660#issuecomment-384062553
-	shouldRetry := activatorutil.RetryStatus(http.StatusServiceUnavailable)
-	backoffSettings := wait.Backoff{
-		Duration: minRetryInterval,
-		Factor:   exponentialBackoffBase,
-		Steps:    maxRetries,
-	}
-	rt := activatorutil.NewRetryRoundTripper(activatorutil.AutoTransport, logger, backoffSettings, shouldRetry)
-	getProbeCount := 0
-	// When network probing is enabled remove the retrying transport
-	// and pass in the retry count for our network probes instead.
-	if *enableNetworkProbe {
-		logger.Info("Enabling network probing for activation.")
-		getProbeCount = maxRetries
-		rt = activatorutil.AutoTransport
-	}
+	rt := activatorutil.AutoTransport
```

Comment:
what's the point of these 2 variables anymore?

Reply:
What variables... ;-)